src/tests/issues.targets

```diff
@@ -920,15 +920,9 @@
         <ExcludeList Include="$(XunitTestBinBase)/Interop/NativeLibrary/Callback/CallbackTests/*">
             <Issue>https://github.com/dotnet/runtimelab/issues/206</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/Interop/PInvoke/Array/MarshalArrayAsField/AsLPArray/AsLPArrayTest/*">
-            <Issue>https://github.com/dotnet/runtimelab/issues/168</Issue>
-        </ExcludeList>
         <ExcludeList Include="$(XunitTestBinBase)/Interop/PInvoke/Array/MarshalArrayAsParam/AsDefault/AsDefaultTest/*">
             <Issue>https://github.com/dotnet/runtimelab/issues/176: VARIANT marshalling</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/Interop/PInvoke/Array/MarshalArrayAsParam/AsLPArray/AsLPArrayTest/*">
```

Comment:
Is this one fixed with this?

I would have expected these lines to go instead because those refer to this issue number:

https://github.com/dotnet/runtime/blob/43ceed5514ca93242ff41f9f1ee53fc108c5dc1b/src/tests/issues.targets#L971-L973

Reply:
This test fully working right now. It was fixed previously mostly fully when I work on VARIANT marshaller. So TypeLoadException was probably just single issue so far which is left.

Unfortunately SafeHandleTests require improvements in SafeHandleMarshaller which is too much for me right now. Also I miss that test because I look for issue description and `Interop/PInvoke/SafeHandles/SafeHandleTests` was not there.